crypto-ffi/Makefile.toml

```diff
@@ -217,7 +219,8 @@ args = [
     "--",
     "-C", "strip=symbols",
     "-l", "static=clang_rt.builtins-x86_64-android",
-    "-L", "${CLANG_RT_DIR}"
+    "-L", "${CLANG_RT_DIR}",
+    "-C", "link-arg=-Wl,-z,max-page-size=16384"
```

Comment:
Why is this needed? Isn't this supposed to be handled automatically by the correct tooling?